src/App.tsx

```diff
@@ -1,56 +1,90 @@
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useRef, useState,
```

Comment:
Do not keep unused variables and imports.
![Screenshot from 2023-12-09 15-44-49](https://github.com/mate-academy/react_autocomplete/assets/33029864/f46293ca-6137-4573-9c4d-6daf10a21af0)